modules/openapi-generator/src/main/resources/JavaVertXWebServer/supportFiles/pom.mustache

```diff
@@ -65,16 +80,15 @@
         <plugins>
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.3</version>
+                <version>${maven-compiler-plugin.version}</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
@slinkydeveloper Is it correct to say that Vert.x 4.x only works with JDK11 or later, right?

Reply:
Nope, it works java 8+

Reply:
Then shall we keep the same value `1.8`?

Reply:
done